webrender/src/frame.rs

```diff
@@ -437,11 +437,6 @@ impl Frame {
                 stacking_context.mix_blend_mode_for_compositing())
         };
 
-        if composition_operations.will_make_invisible() {
-            traversal.skip_current_stacking_context();
```

Comment:
instead of removing the condition, maybe we should leave it in, comment out the inner 2 lines, and add a big loud TODO item?